internal/test_helpers/scenarios/test_bot/script.sh

```
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd $(dirname $0)
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)